src/main/java/me/gaminglounge/estats/listener/GetDamage.java

```diff
@@ -25,22 +27,32 @@ public class GetDamage implements Listener {
 
 
     @EventHandler
-    public void onDamagedbBy(EntityDamageByEntityEvent event){
+    public void onDamagedBy(EntityDamageByEntityEvent event){
 
-        //damageTest(event.getDamage(), event.getEntity().getType(), event.getDamager().getName(),event.getDamageSource().getDamageType());
-        //damageDisplay(event.getDamage(),event.getDamager());
         EStats.INSTANCE.entityName.add_(event.getEntity());
-        if(event.getDamager() instanceof Player & event.getEntity() instanceof LivingEntity){
+        if(event.getDamager() instanceof Player
+        || event.getDamager() instanceof Projectile
+        & event.getEntity() instanceof LivingEntity){
+            double randomOffset = ThreadLocalRandom.current().nextDouble(-0.4, 0.4);
+            double randomLowPitch = ThreadLocalRandom.current().nextDouble(0.56, 0.57);
+            double volume = 0.3;
+            
             event.getEntity().setCustomNameVisible(true);
             Bukkit.getScheduler().runTaskLater(EStats.INSTANCE, ()->EStats.INSTANCE.entityName.remove_(event.getEntity()), 60);
 
-            double randomOffset = ThreadLocalRandom.current().nextDouble(-0.4, 0.4);
-
             Location entityposition = event.getEntity().getLocation();
             TextDisplay counter = (TextDisplay) entityposition.getWorld().spawnEntity(entityposition.add(randomOffset,randomOffset+1,randomOffset), EntityType.TEXT_DISPLAY);
             counter.text(miniMessage.deserialize("<red>-"+String.format("%.1f", event.getDamage())+"</red>"));
             counter.setBillboard(Billboard.VERTICAL);
             counter.setPersistent(false);
+            if(event.getDamager() instanceof Projectile proj)
+                if(proj.getShooter() instanceof Player p)
+                {
+                    p.playSound(p,Sound.ENTITY_EXPERIENCE_ORB_PICKUP ,(float) volume , (float) randomLowPitch);
+                }
+                if(event.getDamager() instanceof Player p){
+                    p.playSound(event.getDamager(),Sound.ENTITY_EXPERIENCE_ORB_PICKUP ,(float) volume , (float) randomLowPitch);
+                }
```

Comment:
Wierd indentation but ok.

Reply:
Not really fixed in https://github.com/GamingLounge-me/EStats/pull/6/commits/4b41a489959254b141a15a9baf330469bd785373 idk what really works well.